whelk-core/src/main/groovy/whelk/JsonLd.groovy

```diff
@@ -584,10 +584,10 @@ class JsonLd {
     }
 
     static List<List<String>> findPaths(Map obj, String key, Set<String> values) {
-        List paths = []
-        new DFS().search(obj, { List path, v ->
+        List<List<String>> paths = []
```

Comment:
paths are not `List<String>`?
for example `['publication', 0, 'hasPart' , 0, 'place', '@type']`

Reply:
Right, good catch!